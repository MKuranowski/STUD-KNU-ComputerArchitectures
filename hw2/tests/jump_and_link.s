addi x10, x0, 20
jalr x1, x10, 0
jal x9, over
setterZ: addi x14, x0, 1
jalr x0, x4, 0
setterA: addi x11, x0, 1
jalr x0, x1, 0
setterB: addi x12, x0, 1
jalr x0, x2, 0
setterC: addi x13, x0, 1
jalr x0, x3, 0
over: jalr x2, x10, 8
jal x3, setterC
jalr x4, x10, -8
lui x31, 912092
addi x31, x31, -273
