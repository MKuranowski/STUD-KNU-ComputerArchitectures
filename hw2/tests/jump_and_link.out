Processor's clock cycles: 16
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 64
x00 = 0
x01 = 8
x02 = 48
x03 = 52
x04 = 56
x05 = 0
x06 = 0
x07 = 0
x08 = 0
x09 = 12
x10 = 20
x11 = 1
x12 = 1
x13 = 1
x14 = 1
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = 0
x21 = 0
x22 = 0
x23 = 0
x24 = 0
x25 = 0
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = -559038737
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
